src/wallet/mod.rs

```diff
@@ -34,14 +42,16 @@ use bitcoin::{
     transaction::Version as TxVersion,
     Amount, Block, ScriptBuf, Sequence, Transaction, TxIn, TxOut, Txid, Witness,
 };
+use error::WalletError;
```

Comment:
Please use std-external-crate convention for imports
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports